Formula/lesspipe.rb

```diff
@@ -33,4 +33,11 @@ def install
     assert File.exist?("homebrew.tar.gz")
     assert_match /file2.txt/, shell_output("tar tvzf homebrew.tar.gz | #{bin}/tarcolor")
   end
+
+  def caveats
+    <<-EOS
+      Append the following to your .bash_profile/.bashrc:
+      export LESSOPEN="|$(brew --prefix)/bin/lesspipe.sh %s" LESS_ADVANCED_PREPROCESSOR=1
```

Comment:
Our current practice for caveats is to use `#{HOMEBREW_PREFIX}` and not `$(brew --prefix)`